docs/content/posts/2024-07-15-release-1.20.0.md

```diff
@@ -0,0 +1,553 @@
+data pipelines, while providing a consistent development experience.
+
+Materialized tables are defined with a query and a data freshness specification. The engine automatically derives the table
+schema and creates a data refresh pipeline to maintain the query result with the requested freshness. Users are relieved from
+the burden of comprehending the concepts and differences between streaming and batch processing, and they do not have to directly
+maintain Flink streaming or batch jobs. All operations are done on Materialized tables, which can significantly accelerate ETL pipeline
+development.
+
+Here is an example to create a materialized table that is constantly refreshed with a data freshness of `3` minutes.
+
+```sql
+-- 1. Create table schema and data refresh pipeline
+CREATE MATERIALIZED TABLE dwd_orders
+(
+ PRIMARY KEY(ds, id) NOT ENFORCED
+)
+PARTITIONED BY (ds)
+FRESHNESS = INTERVAL '3' MINUTE
+AS SELECT 
+ o.ds
+ o.id,
+ o.order_number,
+ o.user_id,
+...
+FROM 
+ orders as o
+ LEFT JOIN products FOR SYSTEM_TIME AS OF proctime() AS prod
+ ON o.product_id = prod.id
+ LEFT JOIN order_pay AS pay
+ ON o.id = pay.order_id and o.ds = pay.ds;
+
+-- 2. Pause the data refresh pipeline
+ALTER MATERIALIZED TABLE dwd_orders SUSPEND;
+
+-- 3. Resume the data refresh pipeline
+ALTER MATERIALIZED TABLE dwd_orders RESUME
+-- Set table option via WITH clause
+WITH(
+ 'sink.parallesim' = '10'
+);
+
+-- Refresh historical partition manually
+ALTER MATERIALIZED TABLE dwd_orders REFRESH PARTITION(ds='20231023');
+```
+
+**More Information**
+* [FLINK-35187](https://issues.apache.org/jira/browse/FLINK-35187)
+* [FLIP-435](https://cwiki.apache.org/confluence/x/HYySEQ)
+* [Materialized Table Overview](https://nightlies.apache.org/flink/flink-docs-release-1.20/docs/dev/table/materialized-table/overview/)
+
+
+## Introduce Catalog-Related Syntax
+
+With the growing adoption of Flink SQL, implementations of Flink's `Catalog` interface play an increasingly important role. Today, Flink features a JDBC and a Hive catalog implementation and other open source projects such as Apache Paimon integrate with this interface as well.
+
+Now in Flink 1.20, you can use the `DQL` syntax to obtain detailed metadata from existing catalogs, and the
+`DDL` syntax to modify metadata such as properties or comment in the specified catalog.
+
+```sql
+Flink SQL> CREATE CATALOG `cat` WITH ('type'='generic_in_memory', 'default-database'='db');
+[INFO] Execute statement succeeded.
+
+Flink SQL> SHOW CREATE CATALOG `cat`;
++---------------------------------------------------------------------------------------------+
+|                                                                                      result |
++---------------------------------------------------------------------------------------------+
+| CREATE CATALOG `cat` WITH (
+  'default-database' = 'db',
+  'type' = 'generic_in_memory'
+)
```

Comment:
could you check the table format? Looks like we missed `|` arounding the content.

Reply:
Looks strange but not related to this PR, LGTM